### PR TITLE
[AVAX] Merge Cortina 3 (v1.10.3)

### PR DESCRIPTION
**Pull request by evlekht**
## Why this should be merged
This PR contains changes from ava-labs/avalanchego between cortina-2 and cortina-3.
### Conflicts
- .github/workflows/test.upgrade.yml
- RELEASES.md
- api/server/server.go
- genesis/beacons.go
- go.mod
- go.sum
- header.yml
- node/node.go
- scripts/build_test.sh
- scripts/constants.sh
- scripts/lint.sh
- scripts/mock.gen.sh
- version/compatibility.json
- version/constants.go
- vms/platformvm/blocks/codec.go
- vms/platformvm/config/config.go
- vms/platformvm/fx/mock_fx.go
- vms/platformvm/txs/codec.go
### Post-merge fixes
- New gh workflows was removed, cause they are not relevant for us.
- `tools/camino-network-runner` dir was excluded from "custom linters" in `scripts/lint.sh`, CNR submodule reverted to previous commit without linter-enforced changes - this way it will have less diff with avax.
- Cause of avax changes, `secp256k1fx.OutputOwners` no longer implements verify.State. Cause of this, `multisig.Alias` no longer implements this interface as well. It now only implements  `snow.ContextInitializable` and `verify.Verifiable`.
- `vms/platformvm/blocks/codec.go` now directly uses `linearcodec.Codec` instead of `codec.Registry` - camino codec was changed accordingly. 
## How this was tested
With already existing unit-tests and e2e tests.

### CODE REVIEW

File: node/node.go
Comment (knikos) [inline node/node.go:1]: for some reason the history of the whole file seems affected. probably cause of camino-node separation and re-merge I guess.

File: header.yml
Comment (knikos) [inline header.yml:1]: is this supposed to be completely removed or just replaced? i see that avax replaced LICENSE.header with license.yml (then renamed to header.yml) during Cortina 2. i know we have our own `LICENSE.header.c4t` but how come did we keep old avax's style and not adapt ours accordingly?

File: node/node.go
Comment (evlekht) [reply node/node.go:1]: yes, thats the case for all files that we removed from caminogo on repo separation

File: header.yml
Comment (evlekht) [reply header.yml:1]: cause of license linter in cortina 2
it doesn't work with two headers to full extent, so we decided to keep old one for now and deal with it later

**Review by knikos — APPROVED** [review]
LGTM

**Pull request merged**